.github/workflows/build-appimage.yml

```diff
@@ -0,0 +1,78 @@
+# This is a basic workflow to help you get started with Actions
+
+name: AppImage CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  pack:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-20.04
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y --no-install-recommends \
+            libfontconfig-dev \
+            libicu-dev \
+            liblzma-dev \
+            liblzo2-dev \
+            libsdl1.2-dev \
+            libsdl2-dev \
+            libxdg-basedir-dev \
+            zlib1g-dev \
+            # EOF
+
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - name: Checkout
+        uses: actions/checkout@v2
+      
+      - name: Build
+        run: |
+          mkdir -p build && cd build
+          cmake .. -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Release
+          make -j`nproc`
+
```

Comment:
Could the existing Ubuntu 20.04 build in `ci-build.yml` produce this as an artifact?

The CPU time might be free to us, but it seems wasteful to run the same compile twice.

Reply:
Yes. I took that workflow as template. The only requirement is to use `/usr` as install prefix. The build type is optional, so you could also have Debug bundles.